src/emoji/RealmEmoji.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import React from 'react';
+import { StyleSheet, Image } from 'react-native';
+import { connect } from 'react-redux';
+import { getAuth } from '../account/accountSelectors';
+import { Auth } from '../types';
+
+const styles = StyleSheet.create({
+  image: {
+    width: 20,
+    height: 20,
+  },
+});
+
+type Props = {
+  auth: Auth,
+  realmEmoji: {},
+  name: string,
+};
+
+class RealmEmoji extends React.PureComponent {
+
+  props: Props;
+
+  render() {
+    const { name, realmEmoji, auth } = this.props;
+
+    return (
+      <Image
+        style={styles.image}
+        source={{ uri: auth.realm.concat(realmEmoji[name].source_url) }}
+      />
+    );
+  }
+}
+
+const mapStateToProps = (state) => ({
```

Comment:
We stopped using mapStateToProps being specified as a separate function, have a look at the other components in `master`.